libfwupd/fwupd-client.c

```diff
@@ -626,6 +626,9 @@ fwupd_client_curl_new(FwupdClient *self, GError **error)
 	(void)curl_easy_setopt(helper->curl, CURLOPT_NOPROGRESS, 0L);
 	(void)curl_easy_setopt(helper->curl, CURLOPT_FOLLOWLOCATION, 1L);
 	(void)curl_easy_setopt(helper->curl, CURLOPT_MAXREDIRS, 5L);
+#if CURL_AT_LEAST_VERSION(7, 71, 0)
+	(void)curl_easy_setopt(helper->curl, CURLOPT_SSL_OPTIONS, CURLSSLOPT_NATIVE_CA);
```

Comment:
I guess you checked this option makes sense outside Windows too?.

Reply:
It doesn't' do anything on !win32 according to the docs. I *think* we can drop the bundled crt too, but wanted to do some real world testing first.